src/veh_interact.cpp

```diff
@@ -494,7 +494,7 @@ bool veh_interact::can_install_part(int msg_width){
                 veh->part_info(p).fuel_type != "muscle")
             {
                 engines++;
-                dif_eng = dif_eng / 2 + 12;
+                dif_eng = dif_eng / 2 + 8;
```

Comment:
This may need revisited but as is, it's probably good enough to hard-cap skills at 10 and begin the rebalance.  Thanks.